### DISCUSSION TITLE
SocialFi App

### DISCUSSION THREAD

**Discussion by oguzhanod** [zkSync-Community-Hub/zksync-developers#151] [General] [answered]
I need some developers to develop a socialFi app on zksync. If we can be successful tvl will be great on zksync

**Comment by MexicanAce** [comment, marked as answer]
Hi @oguzhanod :wave:

Thanks for the enthusiasm! We're closing this Discussion for now, but please feel free to reach out to our Discord for further interest from the community.

Discord: https://join.zksync.dev/
